<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreActiviteTrad</class>
 <widget class="QDialog" name="FenetreActiviteTrad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bOKcancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lTitre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lAjouterAT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Ajouter une Activit√© Traditionnelle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lDuree">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duree :</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="inDateTimeProg">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>90</y>
     <width>112</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="inDuree">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lProgrammation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Programmation : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inTitre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bOKcancel</sender>
   <signal>accepted()</signal>
   <receiver>FenetreActiviteTrad</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bOKcancel</sender>
   <signal>rejected()</signal>
   <receiver>FenetreActiviteTrad</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
